src/main/java/com/mars/portfolio/domain/Portfolio.java

```diff
@@ -0,0 +1,65 @@
+package com.mars.portfolio.domain;
+
+
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "tbl_portfolio")
+@Getter
+public class Portfolio {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "portfolio_id")
+    private Long id;
+
+    private Long memberId;
+
+    @Column(nullable = true)
+    private Long file_id;
```

Comment:
Java에서 필드, 변수 naming 컨벤션은 file_id처럼 snake_case가 아닌 fileId와 같이 camelCase를 이용합니다 
이외에 필드들도 컨벤션에 맞게 변경해주세요!

원활한 협업을 위해 컨벤션에 맞추어 개발하는 걸 습관화해보는 게 좋습니다!
fyi; 추가로 JPA는 camelCase에 맞추어 개발을 해두면 쿼리 실행 시 해당 필드를 DB naming 컨벤션(DB는 snake_case)에 맞추어 변형하여 쿼리를 실행해주기에 camelCase로 작성해도 안전합니다
```suggestion
    private Long fileId;
```